certified-connectors/IronDefense/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2246 @@
+{
+  "securityDefinitions": {
+    "basic": {
+      "type": "basic"
+    }
+  },
+  "security": [
+    {
+      "basic": [],
+      "jwt": []
+    }
+  ],
+  "swagger": "2.0",
+  "info": {
+    "title": "IronNet IronAPI",
```

Comment:
@marcus-defazio Did you gone thru the [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title) ? The title can not contain the words `"API"`.

Reply:
Oops. Changed.